docs/esa/hubble/hubble.rst

```diff
@@ -430,11 +359,115 @@ and a set of criteria to filter the results.
 This will perform a cone search with radius 7 arcmins around the target (defined by
 its coordinates or its name) using the filters defined when executing the function.
 
-This function allows the same parameters than the search criteria (see Section 5).
+This function allows the same parameters than the search criteria (see Section 2).
+
+--------------------------
+5. Getting Hubble products
+--------------------------
+
+.. warning::
+
+   Please bear in mind that the default format to download
+   sets of files has been modified from TAR to ZIP.
 
+After retrieving the metadata, the user can filter the result table and get the rows of interest.
+The most important column is 'observation_id' and it is possible to use it to retrieve all the
+associated files.
+
+In eHST is it possible to download products based on their observation ID (mandatory) and
+a required calibration_level (RAW, CALIBRATED, PRODUCT or AUXILIARY) and/or product type (SCIENCE, PREVIEW, THUMBNAIL or AUXILIARY).
+
+Deprecation Warning: product types PRODUCT, SCIENCE_PRODUCT or POSTCARD are no longer supported. Please modify your scripts accordingly.
```

Comment:
Add a note or some other directive around this to highlight.

Reply:
Notes and warnings included.